charts/reporting-operator/templates/custom-resources/report-queries/pod-volume-request.yaml

```diff
@@ -0,0 +1,120 @@
+    type: string
+    unit: kubernetes_persistentvolume
+  - name: namespace
+    type: string
+    unit: kubernetes_namespace
+  - name: storageclass
+    type: string
+    unit: kubernetes_storageclass
+  - name: volume_request_memory_bytes
+    type: double
+    unit: bytes
+  - name: timeprecision
+    type: double
+    unit: seconds
+  - name: volume_request_memory_byte_seconds
+    type: double
+    unit: byte_seconds
+  - name: timestamp
+    type: timestamp
+    unit: date
+
+  query: |
+      SELECT labels,
+      labels['persistentvolumeclaim'] as persistentvolumeclaim,
+      labels['volumename'] as persistentvolume,
+          labels['namespace'] as namespace,
+          labels['storageclass'] as storageclass,
+          amount as volume_request_memory_bytes,
+          timeprecision,
+          amount * timeprecision as volume_memory_byte_seconds,
+          "timestamp"
+      FROM {| dataSourceTableName "persistentvolumeclaim-request-bytes" |}
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "pod-persistentvolumeclaim-request"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "persistentvolumeclaim-request-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: persistentvolumeclaim
+    type: string
+    unit: kubernetes_persistentvolumeclaim
+  - name: namespace
+    type: string
+    unit: kubernetes_namespace
+  - name: volumename
+    type: string
+    unit: kubernetes_persistentvolume
+  - name: storageclass
+    type: string
+    unit: kubernetes_storageclass
+  - name: timestamp
+    type: timestamp
+    unit: date
+  - name: data_start
+    type: timestamp
+    unit: date
+  - name: data_end
+    type: timestamp
+    unit: date
+  - name: volume_request_memory_byte_seconds
+    type: double
+    unit: byte_seconds
+  inputs:
+    - name: ReportingStart
+    - name: ReportingEnd
+
+  query: |
+    SELECT
+      timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
+      timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
+      persistentvolumeclaim,
+      namespace,
+      volumename,
+      storageclass,
+      min("timestamp") as data_start,
+      max("timestamp") as data_end,
+      sum(volume_request_memory_byte_seconds) as volume_request_memory_byte_seconds
+    FROM {| generationQueryViewName persistentvolumeclaim-request-raw" |}
+    WHERE "timestamp" >= timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prestoTimestamp |}'
+    AND "timestamp" < timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}'
+    GROUP BY volume_request_memory_bytes, namespace, persistentvolume, storageclass
```

Comment:
You don't want to group by on `volume_request_memory_bytes`, you want `persistentvolumeclaim`. `volume_request_memory_bytes` is part of the aggregation, so it can't be in the GROUP BY.

So the whole GROUP BY is:  `GROUP BY persistentvolumeclaim, namespace, persistentvolume, storageclass`.